apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authenticators.tsx

```diff
@@ -357,6 +362,31 @@ export const Authenticators: FunctionComponent<AuthenticatorsPropsInterface> = (
         return AuthenticatorMeta.getAuthenticatorLabels(authenticator.authenticatorId) ?? [];
     };
 
+    /**
+     * Render feature status chip.
+     *
+     * @param authenticator - Authenticator.
+     *
+     * @returns Feature status chip.
+     */
+    const renderFeatureStatusChip = (authenticator: GenericAuthenticatorInterface): ReactElement => {
```

Comment:
Can't we take the `authenticator ` directly, without passing it to the method ?

Reply:
The authenticator is taken by iterating over the `authenticators` list https://github.com/wso2/identity-apps/blob/fc01f0e22bf4c7f6c344573961e097598d017d07/apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authenticators.tsx#L393. So we can't directly take it.